app/main.py

```diff
@@ -806,6 +807,19 @@ def get_owner_email(owner_id):
     else:
         return jsonify({"email": res[0].email})
 
+# Get information of the latest body scaffold for species.
+# This endpoint returns the metadata file path, bucket,
+# dataset id and version which can be used to construct the url
+@app.route("/get_body_scaffold_info/<species>", methods=["GET"])
+def get_body_scaffold_info(species):
+    # Filter to find user based on provided int id
```

Comment:
I find this comment confusing.  You are getting a dataset id from a species name?

Reply:
Oops, it should not be there.